geolocation-API/enabled-by-feature-policy-attribute.https.sub.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/feature-policy/resources/featurepolicy.js"></script>
+<script>
+  "use strict";
+
+  const same_origin_src =
+    "/feature-policy/resources/feature-policy-geolocation.html";
+  const cross_origin_src =
+    "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
+
+  async_test(t => {
+    test_feature_availability(
+      'geolocation',
```

Comment:
Be consistent about " vs '.

```suggestion
      "geolocation",
```